.github/workflows/zap-cache.yml

```diff
@@ -0,0 +1,72 @@
+name: Zap Artifactory Cache
+
+on:
+    workflow_dispatch:
+        inputs:
+            paths:
+                description:
+                    Comma-separated list of paths to zap (e.g.
+                    external/ncd/apps/official/,external/ncd/apps/public-test/source.json)
+                required: true
+                type: string
+
+jobs:
+    zap-cache:
+        runs-on: ubuntu-latest
+        name: Zap Artifactory Cache
+        steps:
+            - uses: actions/github-script@v7
+              env:
+                  ARTIFACTORY_TOKEN: ${{ secrets.COM_NORDICSEMI_FILES_PASSWORD_SWTOOLS_FRONTEND }}
+                  PATHS: ${{ inputs.paths }}
+              with:
+                  script: |
+                      const paths = process.env.PATHS.split(',')
+                          .map(p => p.trim())
+                          .filter(Boolean)
+                          .filter(p => p.startsWith('external/ncd/'))
+                          .filter(p => /^[\w/-]+$/.test(p));
+
+                      if (paths.length === 0) {
+                          throw new Error('No valid paths provided. Paths must start with "external/ncd/" and contain only word characters, slashes, and hyphens.');
+                      }
+
+                      console.log('Paths to process:', paths);
+
+                      const success = [];
+                      const errors = [];
+
+                      for (const path of paths) {
+                          const url = `https://files.nordicsemi.cn/artifactory/swtools-cache/${path}`;
```

Comment:
Maybe I misunderstood or am missing some context, but why only the .cn one?
Aren't there 2 caches?

Reply:
At least currently there is only files.nordicsemi.cn. 

files.nordicsemi.com is the main server (which also goes by the name eu.files.nordicsemi.com).